Sources/TuistLoader/Loaders/CachedManifestLoader.swift

```diff
@@ -312,6 +318,7 @@ private struct Hashes: Equatable, Codable {
     var helpersHash: String?
     var pluginsHash: String?
     var environmentHash: String?
+    var disableSandboxHash: String
```

Comment:
I included this arg in the cached manifest hash, let me know if you want something more robust.